src/libs/API.js

```diff
@@ -929,6 +929,17 @@ function GetCurrencyList() {
     return Mobile_GetConstants({data: ['currencyList']});
 }
 
+/**
+ * @param {object} parameters
+ * @param {string} [parameters.type]
+ * @param {string} [parameters.policyName]
```

Comment:
Capitalize the param types like we do elsewhere?

Reply:
Done! I copied those from another comment, so I have also fixed these.